.travis.yml

```diff
@@ -14,5 +14,5 @@ deploy:
     secure: "JnYRoQqID4nareQqAO2kOkq203FLjVLpN5ncWrsykuttOQK6KJXbuYJZTT65RJzO6iMfnz8YV/fzvL9c18po/pyxtEwTB2Ac9fdwjlHmOXthkt3uxAA7O89yr5kulwmMZ/sCb9QRI00kuA9dE8wEfmyLt5cF11Y5Qw1R8XGSzS+MKpO5nXA6jRg9VCXhkDKfZC3s+4Xm2K35xWwVit07DnAdhrqPW3hyDtSFFVd5cBPtHWzJbg6VNfPx7DRWe26wa/DExzPP9iz0Dv4DWew1rHKcFA5Y7PHxEKaL200lZig4W7Tg4ASpQwZP7fGO+NWPnfX9GZBMYN3z9FzziWzg+Q+DzUsbmobGuGQoIHbpEEYkHcIyb+KwN7J9GiUHqznjHu792aUsRfTasINNMzIXzf8ahfbIYiZfxyriF0elZDbWG9O4Avb15P6tBuOsDYdurEcN3BdLmnepTj2edotVK/7mSMhjVZEWBDhffMpStskt0CqFiUYb9MZgdLDr+cmLCYx9d36RE0Q8nv8oAd5naZ3ccwv+ZJiGBWv5YyhpkrIHQbH185DHV7bFBJkx87ck28t4cC8FKLi+Ry60MBJTD3DT93jdDVmysqV453ipaCX3akYOuLJINmBJHSxU+eRqAqnTY8ZMBjClZBwOxq0Xp7BZoWEGJhvjTvR1xd+RYys="
   on:
     repo: ManiacalLabs/BiblioPixel
-    python: 3.4
+    python: 3.5
```

Comment:
No no... this is because of that pypi issue you ran into that required python 3.4.6 or something which was only ever a source release. Travis doesn't provide 3.4.6 and the issue is fixed in all of 3.5.x 
And remember that this is an sdist release. Python target version doesn't actually matter... Travis will not do the release push unless 3.4 and 3.5 tests succeed. It's just that the commands to release happen on the 3.5 path. I promise, this is only a good thing.